main.py

```diff
@@ -270,12 +271,22 @@ def workers_edit(worker_id):
                 Worker.phone: request.form["phone"] or None,
                 Worker.notes: request.form["notes"],
                 Worker.active: bool(request.form.get("active")),
+                Worker.organizing_dept_id: Department.get(request.form["organizing_dept"] == Department.name).id,
             }
         ).where(Worker.id == worker_id).execute()
         flash("Worker data updated")
 
+    dept_list = [dept.name for dept in Department.select().order_by(Department.name)]
```

Comment:
Again, awkward. Is there a way to get all the names for every Department without looping over all the departments?